src/box/lua/config/instance_config.lua

```diff
@@ -1241,6 +1241,53 @@ return schema.new('instance_config', schema.record({
             validate = feedback_validate,
         }),
     }),
+    flightrec = schema.record({
+        enabled = enterprise_edition(schema.scalar({
+            type = 'boolean',
+            box_cfg = 'flightrec_enabled',
+            default = false,
+        })),
+        logs_size = enterprise_edition(schema.scalar({
+            type = 'integer',
+            box_cfg = 'flightrec_logs_size',
+            default = 10485760,
+        })),
+        logs_max_msg_size = enterprise_edition(schema.scalar({
+            type = 'integer',
+            box_cfg = 'flightrec_logs_max_msg_size',
+            default = 4096,
+        })),
+        logs_log_level = enterprise_edition(schema.scalar({
+            type = 'integer',
+            box_cfg = 'flightrec_logs_log_level',
+            default = 6,
+        })),
```

Comment:
Nit: Shouldn't we list all possible values as in the `log.level` option? AFAIS from the code (`flightrec_check_cfg`), `flightrec_logs_log_level` doesn't support string values and it is OK to don't add them here too. But I would restrict values to 0-7.

Reply:
Thanks, fixed.